Add event details section to homepage

escherlies opened the pull request:
:arrow_upper_right:  #75 

- Add schedule information
- Add meals info
- Add facilities info

<img width="2880" height="2420" alt="Screen Shot 2025-08-02 at 14 31 33" src="https://github.com/user-attachments/assets/f643f5b5-5276-422b-8fc1-3a79a1609e58" />

escherlies committed:
Add event details section to homepage

- Add schedule information
- Add meals info
- Add facilities info

infinisil reviewed (approved):
Let's wait a bit before merging, but it looks great to me!

Nebucatnetzer commented:
IIRC door closing is at either 20:00 or 21:00.

fmehta commented:
Yes, please change the doors closing time to 20h00 on Friday and Saturday, and 17h00 on Sunday so that we have some time to clean up at the end.

escherlies committed:
Update doors closing times in schedule

- Change doors closing from 22:00 daily to specific times per day
- Friday and Saturday now close at 20:00
- Sunday now closes at 17:00
